Resources/Prototypes/_White/Catalog/uplink_catalog.yml

```
@@ -81,3 +81,67 @@
+      tags:
+      - NukeOpsUplink
+
+- type: listing
+  id: UplinkThermalGogglesNukie
+  name: uplink-thermal-vision-name
+  description: uplink-thermal-vision-desc
+  productEntity: ClothingEyesThermalVisionGogglesNukie
+  cost:
+    Telecrystal: 3
+  categories:
+  - UplinkArmor
```

yamllint

[warning] wrong indentation: expected 4 but found 2

(indentation)